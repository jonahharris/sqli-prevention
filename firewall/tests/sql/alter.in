>>> Sada testovacich SQL dotazu: alter
ALTER DATABASE CHARACTER SET = 'utf8'  COLLATE = 'utf8_bin';
ALTER SCHEMA DEFAULT CHARACTER SET = 'utf8'  COLLATE = 'utf8_bin';
ALTER DATABASE test CHARACTER SET = 'utf8'  COLLATE = 'utf8_bin';
ALTER SCHEMA test CHARACTER SET = 'utf8'  COLLATE = 'utf8_bin';
ALTER DATABASE `#mysql50#a-b-c` UPGRADE DATA DIRECTORY NAME;
ALTER SCHEMA test UPGRADE DATA DIRECTORY NAME;
