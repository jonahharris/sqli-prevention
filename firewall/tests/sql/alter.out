>>> Sada testovacich SQL dotazu: alter
STRING '***';CHARACTER SET 0 4;STRING '***';COLLATE 0 4;DATABASE THIS;ALTER DATABASE;STMT;1;0;
STRING '***';CHARACTER SET 1 4;STRING '***';COLLATE 0 4;DATABASE THIS;ALTER DATABASE;STMT;1;0;
STRING '***';CHARACTER SET 0 4;STRING '***';COLLATE 0 4;DATABASE test;ALTER DATABASE;STMT;1;0;
STRING '***';CHARACTER SET 0 4;STRING '***';COLLATE 0 4;DATABASE test;ALTER DATABASE;STMT;1;0;
DATABASE #mysql50#a-b-c;ALTER DATABASE UPGRADE;STMT;1;0;
DATABASE test;ALTER DATABASE UPGRADE;STMT;1;0;
